### PR TITLE
Update responsibilities & key skills

### PR DESCRIPTION
**Pull request by cktamucalgary**
For the project team and resources section.

### CODE REVIEW

File: proposal-template.md [hunk 1]
```diff
@@ -116,35 +116,35 @@ Name | Lucas Hossack
...
-**Key Skills** | Programming, algorithms, debugging
+**Responsibilities** | High level architecture design. Set up the framework and ensure that all members meet the set programming style and standards. Help members to integrate their contributions.
+**Key Skills** | Programming, algorithms, debugging, communication
 
 Name | Deanna Wing Yan Ip
 ------------ | -------------
 **Organization** | RouteScouts
 **Role** | Project Manager
-**Responsibilities** | Manage deadlines, ensure deliverables are met
+**Responsibilities** | Manage deadlines and ensure deliverables are met. Follow up with group members when completed work is behind schedule or deliverables are not up to standards.
```
Comment (stynaa) [inline proposal-template.md]: completed should be changed to assigned

File: proposal-template.md [hunk 2]
```diff
@@ -116,35 +116,35 @@ Name | Lucas Hossack
...
+**Responsibilities** | Create requirements for the UI design that are relevant to the users and their tasks. Ensure functionality of UI meets the end user's specifications and that the UI is intuitive.
 **Key Skills** | Programming, design, databases
 
 Name | Joan Ching Kwan Tam
 ------------ | -------------
 **Organization** | RouteScouts
 **Role** | QA Lead
-**Responsibilities** | Test changes to the code
-**Key Skills** | Writing tests, testing tests, running tests
+**Responsibilities** | Test that changes to the code do not change the core functionality of the project, or do not break the project.
```
Comment (stynaa) [inline proposal-template.md]: ", or do not " can be simplified to " or "

File: proposal-template.md (hunk 1 above)
Comment (cktamucalgary) [reply proposal-template.md]: Will do!

File: proposal-template.md (hunk 2 above)
Comment (cktamucalgary) [reply proposal-template.md]: Ok

**Pull request merged**